packages/web.service/src/modules/user/router.ts

```diff
@@ -5,15 +5,27 @@ import { StatusCodes } from 'http-status-codes';
 import { logger } from 'logger';
 import { handleValidationErrors } from 'services/handleValidationErrors';
 
-import { createUserAction, getUserByEmailAction } from './actions';
+import { createUserAction, getUserByEmailAction, logoutUserAction } from './actions';
 import { EmailAlreadyExistsError } from './errors';
+import type { UserDocument } from './schema';
 import { UserRoles } from './schema';
 
 const userLogger = logger.getLogger('router.user');
 
 const allowedRoles = [UserRoles.Admin, UserRoles.Manager];
 
 export const userRouter = Router()
+  .post('/user/logout', (req, res) => {
+    const { id } = req.user as UserDocument;
+    logoutUserAction(id)
+      .then(() => {
+        res.status(StatusCodes.ACCEPTED).send();
```

Comment:
```suggestion
        res.send();
```
Status code 202 Accepted is not correct.
https://httpstatuses.com/202#:~:text=The%20request%20has%20been%20accepted,code%20from%20an%20asynchronous%20operation.